pages/ccf-3d-reference-library.html

```diff
@@ -119,47 +97,51 @@ <h4 class="mt-5">Reference Organs</h4>
                                 <ul class="nav nav-tabs bg-light" id="myTab" role="tablist">
                                     <li class="nav-item">
                                         <a class="nav-link active" id="bone-tab" data-toggle="tab" href="#bone"

```

Comment:
A quick local check found that adding `text-align: center;` to the nav-link css definition does the trick of centering things.